### ISSUE TITLE
Interference with the styles in the NXRM screens

### ISSUE THREAD

**Issue by Blacktiger** [groupe-edf/nexus-report-plugin#8]
I'm assuming this is related to having two versions of the RSC library installed and the stylesheets from both versions exposed, but installing this plugin causes problems with the logging configuration screen.

![image](https://user-images.githubusercontent.com/86939/95228320-cd0b8200-07c4-11eb-96fd-42cc7b794031.png)

**Comment by mat1e** [comment]
@Blacktiger, I developed the plugin on the version 3.24.0-02 of OSS Nexus Repository and the version of @sonatype/react-shared-components may differs on more recent version of the Nexus.
I had difficulty to build the plugin without checking out all of nexus-public. For this reason, I integrated this library in the plugin as suggested in [DX-112](https://issues.sonatype.org/browse/DX-112). 
Maybe it wasn't a good idea, but I don't see any other solution to use the "standard" nexus react components. Or I have to make my own components with their style but in this case the UI of the plugin may differs of the rest of the Nexus repository.
